src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -69,47 +64,52 @@ public ImportCommand(Path importCsvFilePath) {
         JsonUserPrefsStorage userPrefStorage =
             new JsonUserPrefsStorage(projectRootPath.resolve("config.json"));
         this.storage = new StorageManager(jsonStorage, userPrefStorage);
-        haveDuplicatePersons = false;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         requireNonNull(importCsvFilePath);
         requireNonNull(storage);
-        readCsvFile(model);
-        return new CommandResult(MESSAGE_SUCCESS);
+        int studentsImported = readCsvFile(model);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, studentsImported));
     }
 
     /*
      * Reads the CSV file and imports the data into the address book
      */
-    private void readCsvFile(Model model) throws CommandException {
+    private int readCsvFile(Model model) throws CommandException {
```

Comment:
Actually do you think its better to split this method into two, one for reading, one for getting tihe mportedStudent

Reply:
I think this would cause unnecessary code and duplicate code, thus it wouldn't be beneficial